tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "module": "ES2020",
         "lib": [
             "ES2020",
-            "webworker"
+            "dom"
```

Comment:
What's the reason for this change? Is the library interacting with any `dom` elements?